source/jormungandr/jormungandr/street_network/asgard.py

```diff
@@ -52,6 +53,12 @@ def __init__(
         self.timeout = timeout
         self._sockets = queue.Queue()
 
+        self.breaker = pybreaker.CircuitBreaker(
+            fail_max=app.config['CIRCUIT_BREAKER_MAX_ASGARD_FAIL'],
```

Comment:
No possibility to pass fail_max and reset_timeout in kwargs? (as it is done on other connectors)

Reply:
it's done the same way as other connectors :) geovelo/here

Reply:
this is possible. since every connector must use pybreaker, we can put a breaker in the base class `AbstractStreetNetworkService`, but it will impact all street_network connectors... i prefer a refactoring in another PR